src/itest/resources/json/input/psc_statement_legacy.json

```diff
@@ -0,0 +1,21 @@
+{
+  "_id" : "<id>",
+  "created" : {
+    "at" : ISODate("2014-08-29T00:00:00.000Z"),
+  },
+  "company_number" : "<company_number>",
+  "updated" : {
+    "at" : ISODate("2014-08-29T00:00:00.000Z"),
+  },
+  "psc_statement_id_raw" : "3000000002",
```

Comment:
This should be the legacy field `psc_statement_id` so that we are testing that we can update documents which have the legacy field set, as will be the case in live.